build.gradle

```diff
@@ -94,14 +94,10 @@ allprojects {
     }
   }
 }
-
 subprojects {
   dependencies {
     spinnaker.group('spockBase')
-    compile spinnaker.dependency('groovy')
-    compile spinnaker.dependency('guava')
-    compile spinnaker.dependency('slf4jApi')
-    testRuntime spinnaker.dependency('slf4jSimple')
+    testCompile spinnaker.dependency('groovy')
```

Comment:
we would get some version of groovy as a transitive dependency of spock but not explicitly the version we use elsewhere.

most of the other modules have groovy in compile scope explicitly, but this doesn't (intending it to be java-only)